awscli/customizations/s3events.py

```diff
@@ -57,6 +65,46 @@ def replace_event_stream_docs(help_command, **kwargs):
               "object content is written to the specified outfile.\n")
 
 
+def document_expires_string(help_command, **kwargs):
+    doc = help_command.doc
+    caught_index_error = False
+    popped = []
+    current = ''
+    while current != 'Expires -> (timestamp)':
+        try:
+            current = doc.pop_write()
+            popped.append(current)
+        except IndexError:
+            # Do nothing if Expires is not in the modeled response
+            caught_index_error = True
+            break
+
+    # Only inject the deprecation notice if the Expires timestamp arg was found in the document
+    if not caught_index_error:
+        # Put back the expires field and description
+        doc.push_write(popped.pop())
+        doc.push_write(popped.pop())
+        doc.push_write(popped.pop())
+        doc.push_write(popped.pop())
+
+        doc.push_write('\n\n\n' + doc.style.spaces())
+        doc.push_write('.. note::')
```

Comment:
I chose note mostly to stay consistent with the current [botocore docs](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3/client/get_object.html). Another alternative considered is Warning, which may semantically be more fitting (?)